protocol/chat.go

```diff
@@ -40,6 +42,23 @@ const (
 	FirstMessageTimestampNoMessage = 1
 )
 
+const (
+	MuteFor1MinDuration   = time.Minute / time.Second
+	MuteFor15MinsDuration = 15 * time.Minute / time.Second
+	MuteFor1HrsDuration   = time.Hour / time.Second
+	MuteFor8HrsDuration   = 8 * time.Hour / time.Second
+	MuteFor1WeekDuration  = 7 * 24 * time.Hour / time.Second
+)
+
+const (
+	MuteFor15Mins requests.MutingVariation = iota + 1
+	MuteFor1Hrs
```

Comment:
Nitpick on my part, but the abbreviated unit for hour is just `h` and `1 hour` is on the singular form. So the constant could be `MuteFor1H` (weird to me), or `MuteFor1Hour` (the word is longer, but it's also following the same name used by `time.Hour`).

It doesn't matter, just sharing this tiny little thing.